Resources/doc/filters.rst

```diff
@@ -263,7 +263,7 @@ A simple example showing how to change the filter configuration dynamically.
             $this->cacheManager->store($filteredBinary, $path, $filter);
         }
 
-        return new RedirectResponse($this->cacheManager->resolve($path, $filter), 301);
+        return new RedirectResponse($this->cacheManager->resolve($path, $filter), Response::HTTP_MOVED_PERMANENTLY);
```

Comment:
please add a ``note`` here to say that for Symfony 2.3 devs should use ``301`` instead of the constant.